src/Engine/Manager/RenderManager.cpp

```diff
@@ -337,15 +405,15 @@ Component::Animation* RenderManager::CreateAnimation() {
 
 Component::Animation* RenderManager::CreateAnimation(const Json::Value& node) {
     Component::Animation* animation = animations.Create();
-    
+
     // Load values from Json node.
     std::string name = node.get("riggedModel", "").asString();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Variable 'name' is assigned a value that is never used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunreadVariable)